x-pack/plugins/uptime/public/lib/alert_types/tls_legacy.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { AlertTypeModel } from '../../../../triggers_actions_ui/public';
+import { CLIENT_ALERT_TYPES } from '../../../common/constants/alerts';
+import { TlsTranslationsLegacy } from './translations';
+import { AlertTypeInitializer } from '.';
+
+const { defaultActionMessage, description } = TlsTranslationsLegacy;
+const TLSAlert = React.lazy(() => import('./lazy_wrapper/tls_alert'));
+export const initTlsLegacyAlertType: AlertTypeInitializer = ({
+  core,
+  plugins,
+}): AlertTypeModel => ({
+  id: CLIENT_ALERT_TYPES.TLS_LEGACY,
+  iconClass: 'uptimeApp',
+  documentationUrl(docLinks) {
+    return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/uptime/${docLinks.DOC_LINK_VERSION}/uptime-alerting.html#_tls_alerts`;
```

Comment:
Please update relevant docs as well especially for message , otherwise create a follow up issue for docs team.

Reply:
How should we handle the deprecation of the other rule type in the docs?